src/checkers/inference/model/serialization/ToStringSerializer.java

```diff
@@ -182,16 +182,19 @@ public String serialize(ConstantSlot slot) {
         stringBuilder.append("( ");
         stringBuilder.append(slot.getId());
         stringBuilder.append(" => ");
-        String fullAnno = slot.getValue().toString();
-
+        String fullAnno = slot.getValue().toString().substring(1);
+        DeclaredType annoType = slot.getValue().getAnnotationType();
+        
         int index = fullAnno.lastIndexOf('.');
         if (index > -1) {
-            stringBuilder.append("@");
-            stringBuilder.append(fullAnno.substring(index + 1, fullAnno.length()));
+            String fullAnnoNoArg = annoType.toString();
+            String argName = fullAnno.substring(fullAnnoNoArg.length());
```

Comment:
"argName" might be a bit misleading - isn't this all annotation arguments, including the parentheses? So if fullAnno is "@a.b.Anno(key="value", also=55)" then argName is "(key="value", also=55)"